data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+			`	After drawing in a deep breath she stands up straighter and gestures to someone some distance away. "Captain, I'm sorry, but my colleague Julie and I are in dire need of urgent transport. Can you help?"`
+			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Where do you need to go?"`
+				`	"Why is it so urgent?"`
+			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of returning to Valhalla for several important business meetings but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Who's your employer?"`
+			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually but at the moment, indirectly, Deep Security."`
+			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"I'm sorry, but I really can't help you right now."`
+					decline
+			label sure
+			`	"Lead right on then," says Julie.`
+				accept
+	on complete
+		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
+		payment 350000
+		set "deep security consultants"
+		conversation
+			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring you <payment>, she points out how you can change your settings with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
+			`	"Thank you Captain <last>, you've been most helpful. Do keep an eye out in the spaceports around here though, Deep Security is notorious for keeping us longer than we booked for. There's often someone stuck."`
+
+
+phrase "stranded consultant pickup"
+	word
+		`In accordance with the lodging request, you`
+		`Despite the short notice, you`
+	word
+		` `
+	word
+		`meet the consultants at their hotel accommodation`
+		`collect the consultants from the security station`
+		`fish the consultants out of a seedy bar`
+		`rescue the consultants from some overbearing companions`
+	word
+		` `
+	word
+		`in a quiet part of the spaceport`
+		`on the other side of town`
+		`in a raucous quarter just outside the port`
+	word
+		` `
+	word
+		`and escort them on board.`
+
+phrase "stranded consultant dropoff payment"
+	word
+		`Your consultant passengers thank you as you deliver them to their destination.`
```

Comment:
```suggestion
		`Your passengers thank you as you deliver them to their destination.`
```

(emphasizing that they are consultants seems unnecessary here)

Reply:
What about "The consultants thank you" instead?

Reply:
That would be fine.